mta/entities/__init__.py

```diff
@@ -55,20 +57,55 @@ def is_empty(self):
 
     @property
     def is_displayed(self):
-        return (
-            "RED HAT APPLICATION MIGRATION TOOLKIT" in self.header.text
-            and self.header.is_displayed
-            and self.help.is_displayed
+        return self.header.is_displayed and self.help.is_displayed
+
+
+class AllProjectView(BaseLoggedInPage):
+    """This view represent Project All View"""
+
+    title = Text(".//div[contains(@class, 'projects-bar')]/h1")
+    search = Input(".//input[`contains(@name, 'searchValue')]")
+    # TODO: add custom sort widget
+
+    projects = ProjectList(locator=".//div[contains(@class, 'projects-list')]")
+    new_project_button = Button("New Project")
+
+    @View.nested
+    class no_matches(View):  # noqa
+        """After search if no match found"""
+
+        text = Text(".//div[contains(@class, 'no-matches')]")
+        remove = Text(".//div[contains(@class, 'no-matches')]/a")
+
+    def clear_search(self):
+        """Clear search"""
+        if self.search.value:
+            self.search.fill("")
+
+    @property
+    def is_displayed(self):
+        return self.is_empty or (
+            self.new_project_button.is_displayed and self.title.text == "Projects"
```

Comment:
Do you want this changes in rebranding PR?

Reply:
I was working on this and then rebranding came up .. I will keep this for separate PR .